Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -399,10 +417,16 @@ public bool TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
             {
                 using (TryUpdateQueryBufferForLayerMaskPerfMarker.Auto())
                 {
+                    float radius;
```

Comment:
I think one line if/else in this codebase uses braces, would be good to maintain consistency.